### ISSUE TITLE
BUG : Amount in currency conversion "fr" / JPY

### ISSUE THREAD

**Issue by Bryce-Colton** [kwn/number-to-words#160]
We use this package in production and we found an issue with it.

Here the tested code : 

```
(new NumberToWords())
                ->getCurrencyTransformer('fr')
                ->toWords(1000, 'JPY')
            ;
```

Here the raised error : "**Array to string conversion**"

Notice that it works with 100 => "un yen"

But not with 101 for instance => "**Undefined offset: 1**"

Can you fix it, please ?

As a complementary remark, it will be a great addition to have a method that return the supported combinaison between Currency and Langage.

Best regards.

**Comment by kwn** [comment]
Hi, thanks for flagging the issue. What version of PHP are you using?

**Comment by Bryce-Colton** [comment]
Hi, thanks for your consideration.
We are currently using PHP 7.4

Here is the unit test that I wrote to test this package :

[NumberToWordsTest.txt](https://github.com/kwn/number-to-words/files/12501423/NumberToWordsTest.txt)

**Comment by kwn** [comment]
Thanks. Indeed there was a bug. Thanks for spotting and reporting. It's fixed in https://github.com/kwn/number-to-words/pull/163

Will release it soon.

**Comment by kwn** [comment]
Fixed in `2.7.2`

**Comment by Bryce-Colton** [comment]
Awesome news 🙂 !

Thank you for maintaining well this repo !

I will test the new release on next Monday.
